src/components/ChatLog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const entryComponents = props.entries.map((entry) => {
+    return (
+      <div className="chat-log" key={entry.id}>
+        <ChatEntry
```

Comment:
The warning React is giving you is because it wants a `key` prop:

```suggestion
        <ChatEntry
          key={entry.id}
```

React uses this to optimize re-rendering (and it doesn't matter like 95% of the time).